clients/privacy-center/config/config.json

```diff
@@ -3,8 +3,6 @@
   "description": "When you use our services, you’re trusting us with your information. We understand this is a big responsibility and work hard to protect your information and put you in control.",
   "description_subtext": [],
   "addendum": [],
-  "server_url_development": "http://localhost:8080/api/v1",
-  "server_url_production": "http://localhost:8080/api/v1",
```

Comment:
These aren't part of the config anymore, and instead should be specified as an ENV var like `FIDES_PRIVACY_CENTER__API_URL=https://fides.ethyca.com` 👍 

I still have some magic code in the server-environment.ts file to support them for backwards compat though.